src/include/rho/FixedVector.hpp

```diff
@@ -222,9 +222,8 @@ namespace rho {
 	// Virtual function of GCNode:
 	void detachReferents() override;
     private:
-	T* m_data;  // pointer to the vector's data block.
```

Comment:
Unfortunately, we're going to need this pointer later on, as there are cases where we'll want to store the data somewhere other than the end of this object.

Can you benchmark this code and see how big of a benefit it provides?  If it's significant, then it may make sense to push it through anyway and create a slightly different class for those cases.